Frontend/ui-customer/src/Helpers/Utils.js

```diff
@@ -463,3 +474,37 @@ export const getPixels = function (){
   }
 }
 
+export const getDateAfterNDays = function (n) {
+  const today = new Date();
+  const dateAfterNDays = new Date(today);
+  dateAfterNDays.setDate(today.getDate() + n);
+  const year = dateAfterNDays.getFullYear();
+  const month = String(dateAfterNDays.getMonth() + 1).padStart(2, "0");
+  const day = String(dateAfterNDays.getDate()).padStart(2, "0");
+  const result = `${day}/${month}/${year}`;
+  return `${day}/${month}/${year}`;
+}
+
+export const getDateAfterNDaysv2 = function (n) {
```

Comment:
Why is this required? this is same as `getDateAfterNDays`

Reply:
the return format of `getDateAfterNDays` and `getDateAfterNDaysv2` are different

Reply:
however the getDateAfterNDays is not used.. So removing it from ui-customer